### DISCUSSION TITLE
Why <picture> tag fixes the warning on the <img> tag

### DISCUSSION THREAD

**Discussion by 1alexvash** [vercel/next.js#40214] [Help] [answered]
In the NextJS we have this warning:
```Do not use `<img>` element. Use `<Image />` from `next/image` instead. See: https://nextjs.org/docs/messages/no-img-element```

It prohibits us from using images which can cause the issue with layout shifting as I understand

![animation](https://user-images.githubusercontent.com/46556479/188327408-0a1f0f6a-8a86-46f3-8e42-9d1f4eeb1933.gif)
Why if I just wrap the element with the picture tag, the error magically is gone?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is not an error, it is a warning as you point out first. Therefore, it is something to balance, and as you are correctly doing now, try to understand.

The ESLINT rule behind this is actually been modified, improved, to behave as you've found. According to: https://github.com/vercel/next.js/issues/37504:

> Linting throws a warning of @next/next/no-img-element when an `<img>` tag is inside a `<picture>` tag.
> `<Image>` cannot do all the things that `<picture>` can and so using the `<img>` is a legitimate use-case.
> Therefore this warning should not be thrown.

So the explanation is that there's indeed a use case for when `picture > img` combinations can accomplish things that a single `next/image` can't, for those use cases the warning is silenced.
